### ISSUE TITLE
Miner just stops when mining YescryptR8g and others on Zergpool

### ISSUE THREAD

**Issue by RainbowMiner** [rplant8/cpuminer-opt-rplant#7]
.. like the title says. Miner starts but returns to command line quickly:
```
[2020-09-23 08:48:19] Extranonce subscribe: YES
[2020-09-23 08:48:19] API listnening to 127.0.0.1:33333
[2020-09-23 08:48:19] Stratum connect yescryptR8G.mine.zergpool.com:6324
[2020-09-23 08:48:19] Binding thread 0 to mask 000000000000000f
[2020-09-23 08:48:19]* Binding thread 2 to mask 000000000000000f
 [2020-09-23 08:48:19] Trying 103.249.70.7:6324...
 Binding thread 1 to mask 000000000000000f*
[2020-09-23 08:48:19] 4 miner threads started, using 'yescryptr8g' algorithm.
TCP_NODELAY set
[2020-09-23 08:48:19] Binding thread 3 to mask 000000000000000f
* Connected to yescryptR8G.mine.zergpool.com (103.249.70.7) port 6324 (#0)
* Connection #0 to host yescryptR8G.mine.zergpool.com left intact
[2020-09-23 08:48:19] > {"id": 1, "method": "mining.subscribe", "params": ["cpuminer-opt/4.5.16W-sse42"]}
[2020-09-23 08:48:20] < {"id":1,"result":[[["mining.set_difficulty","1.2"],["mining.notify","59e94ad204ce664552aff2c8212e919a"]],"8100020d",4],"error":null}
[2020-09-23 08:48:20] Stratum session id: 59e94ad204ce664552aff2c8212e919a
[2020-09-23 08:48:20] > {"id": 2, "method": "mining.authorize", "params": ["3DxRETpBoXKrEBQxFb2HsPmG6apxHmKmUx", "rbmcpu,c=BTC,mc=KOTO"]}
[2020-09-23 08:48:20] < {"id":2,"result":true,"error":null}
[2020-09-23 08:48:20] > {"id": 3, "method": "mining.extranonce.subscribe", "params": []}
[2020-09-23 08:48:21] < {"id":null,"method":"mining.set_difficulty","params":[1]}
[2020-09-23 08:48:21] Stratum connection established
[2020-09-23 08:48:21] < {"id":null,"method":"mining.notify","params":["3ddd","140d3853d1a9f963b7d2e003dd7da24b82bb3b171b062a8d2ebcc928f500f7cb","040000800ae52390010000000000000000000000000000000000000000000000000000000000000000ffffffff1f03c6321604a4ef6a5f08","7a657267706f6f6c2e636f6d00ffffffff0285441521010000001976a9143363229d64cbfcb76643670d1934da5d8138b27588ac80d5f0080000000017a914ac51c292ed3db86651049f3c5d727183f17323ea8700000000000000000000000000000000000000",["fee30ccf0194ffa1b849e82c05335ccede66aa966d48cb2dcf4065dd6097c00d","baf388c0359c510d6d8dc9cf8b59a6eba0ab251d275286ebfa97337b93a3475e"],"00000005","1d40a481","5f6aefa4",true,"8f858e4e11eddcb0ddb74785907e1bf6faf37253b2c73fb5454e00ae5f0780b2"]}
[2020-09-23 08:48:21] DEBUG: job_id='3ddd' extranonce2=00000000 ntime=5f6aefa4
[2020-09-23 08:48:21] New stratum difficulty 1
[2020-09-23 08:48:21] yescryptR8G.mine.zergpool.com:6324 yescryptr8g block 1454790, job 3ddd, network diff 0.01546944
[2020-09-23 08:48:21] < {"id":3,"result":true,"error":null}

C:\RainbowMiner\Bin\CPU-Rplant>
```

The same commandline works fine on miner release 4.5.11w

**Comment by rplant8** [comment]
what processor? does the sse2 version work?

**Comment by RainbowMiner** [comment]
I have tested the miner on an Intel(R) Pentium(R) CPU G4560 @ 3.50GHz
Yes, the sse2 version crashes as well.

Here is my command line:
```
cpuminer-sse2.exe -b 33300 -a yescryptr8g -o stratum+tcp://yescryptR8G.mine.zergpool.com:6324 -u 3DxRETpBoXKrEBQxFb2HsPmG6apxHmKmUx -p rbmcpu,c=BTC,mc=KOTO -t 3 --cpu-affinity 0x0007
```

**Comment by RainbowMiner** [comment]
This issue is not solved with new release v4.5.17 - still crashing/return to commandline like described above.

**Comment by RainbowMiner** [comment]
Sorry, new release v4.5.18 still doesn't solve that problem.

**Comment by rplant8** [comment]
windows 10 is evil. everything works on win 7. I can't figure out what the problem is yet

**Comment by rplant8** [comment]
fixed in 4.5.20

**Comment by RainbowMiner** [comment]
Thank you for the fix. I can confirm everything runs fine, now.
